src/client/sse-manager.test.ts

```
@@ -45,4 +45,44 @@ describe('SSEManager', () => {
+      // Track native listeners like a real EventSource would
+      const nativeListeners = new Map<string, Set<Function>>();
+      mockEventSource.addEventListener.mockImplementation((name: string, fn: Function) => {
+        if (!nativeListeners.has(name)) nativeListeners.set(name, new Set());
+        nativeListeners.get(name)!.add(fn);
+      });
+      mockEventSource.removeEventListener.mockImplementation((name: string, fn: Function) => {
+        nativeListeners.get(name)?.delete(fn);
+      });
+
+      const dispatch = (eventName: string, data: any) => {
+        nativeListeners.get(eventName)?.forEach((fn) => fn(data));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)